samples/notebooks/python/08-native-function-inline.ipynb

```diff
@@ -0,0 +1,685 @@
+{
+  "cells": [
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "id": "3c93ac5b",
+      "metadata": {},
+      "source": [
+        "# Running Native Functions"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "ebcabb91",
+      "metadata": {},
+      "source": []
+    },
+    {
+      "attachments": {},
+      "cell_type": "markdown",
+      "id": "40201641",
+      "metadata": {},
+      "source": [
+        "Two of the previous notebooks showed how to [execute semantic functions inline](./03-running-semantic-function-inline.ipynb) and how to [run prompts from a file](./02-running-prompts-from-file.ipynb).\n",
```

Comment:
03-running-semantic-function-inline.ipynb
There is a file name changed. Need to update here.